emperor/support_files/js/draw.js

```diff
@@ -295,11 +299,80 @@ define(['underscore', 'three', 'jquery'], function(_, THREE, $) {
     // the line will contain the two vertices and the described material
     // we increase the number of points to have a smoother transition on
     // edges i. e. where the trajectory changes the direction it is going
-    lineGeometry = new THREE.TubeGeometry(path, (points.length - 1) * 3, radius,
```

Comment:
^ I think we've stopped using the `drawTrajectoryLine` function in the code-base if that's the case can you delete that function and all other associated tests?

Reply:
Done